kohttp/src/test/kotlin/io/github/rybalkinsd/kohttp/client/ClientBuilderTest.kt

```diff
@@ -120,4 +125,42 @@ class ClientBuilderTest {
             }
     }
 
+    @Test
+    fun `client with ssl material`() {
+        val keyStorePath = "identity.jsk"
+        val trustStorePath = "truststore.jks"
+
+        val keyStorePassword = "secret".toCharArray()
+        val trustStorePassword = "secret".toCharArray()
+
+        val keyStore = KeyStore.getInstance(KeyStore.getDefaultType())
+        val trustStore = KeyStore.getInstance(KeyStore.getDefaultType())
+
+        ClientBuilderTest::class.java.classLoader.getResourceAsStream(keyStorePath).use { keyStoreInputStream ->
+            keyStore.load(keyStoreInputStream, keyStorePassword)
+        }
+
+        ClientBuilderTest::class.java.classLoader.getResourceAsStream(trustStorePath).use { trustStoreInputStream ->
+            trustStore.load(trustStoreInputStream, trustStorePassword)
+        }
+
+        val keyManagerFactory: KeyManagerFactory = KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm())
+        keyManagerFactory.init(keyStore, keyStorePassword)
+        val keyManagers: Array<KeyManager> = keyManagerFactory.keyManagers
+
+        val trustManagerFactory: TrustManagerFactory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm())
+        trustManagerFactory.init(trustStore)
+        val trustManagers: Array<TrustManager> = trustManagerFactory.trustManagers
+
+        val sslContext = SSLContext.getInstance("TLSv1.2")
+        sslContext.init(keyManagers, trustManagers, SecureRandom())
+
+        assertThatCode {
```

Comment:
Wow! That's looks great, need to use same assert technique checking exceptions in other tests